content/help/ops/security/repairing-citadel/index.md

```diff
@@ -5,6 +5,15 @@ weight: 10
 keywords: [security,citadel,ops]
 ---
 
+Istio currently supports only a **single Citadel instance**. Running multiple Citadel instances may introduce
+race conditions and can **lead to system outages**.
+
+Citadel is not a critical data plane component. The default workload certificate lifetime is 3
```

Comment:
break this run-on into separate sentences.  If you feel the urge to type "and" try a `.` instead :)

` The default worklaod certificate lifetime is 3 months.  Certificates will be rotated by CIadel before they expire.  If Citadel is disabled for short maintenance periods, existing mutaul TLS traffic will not be affected.

Reply:
done.